js/cats.js

```diff
@@ -0,0 +1,56 @@
+var CATS = {
+    init: function(callback) {
+        var self = this;
+
+        // First-time running code after installation, set "CATS" to disabled by default
+        // Opening up installed page, which will determine,
+        // whether CATS should be enabled or not
+        if (self.isEnabled() === undefined && sessionstorage_get("installed")) {
+            sessionstorage_set("installed"); // present with a new user
+
+            var installURL = "https://catblock.tk/installed.html";
```

Comment:
@kpeckett When a user installs CatBlock from the special link, some data to localStorage will be saved (project-cats will be saved as true).
After we open a post-installation URL, CatBlock will try to get `project-cats` data from localStorage.

When CB detects there is not such an entry, it won't enable "Project CATS" mode. When there is, it will send a message to CatBlock to enable "Project CATS" mode.